deploy/global_config.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
Is having two global_config's ideal?

I would rather we standardise on having one global_config.py in the root, and make all the values configurable through environment variables. That way, no-one should have to edit it.

Reply:
Best approach would be to make them all configurable, scrapping the global_config all together. The idea here is to not rebuild every time global config changes.

Reply:
On that we agree!